<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit_result</class>
 <widget class="QWidget" name="Credit_result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>475</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>475</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>461</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLabel" name="label_m_res">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:20px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_over_res">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:20px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_all_res">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:20px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_all">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:20px;
}</string>
      </property>
      <property name="text">
       <string>Total payment</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_over">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:20px;
}</string>
      </property>
      <property name="text">
       <string>Overpayment</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_m">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:20px;
}</string>
      </property>
      <property name="text">
       <string>Montly payment</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
